docker-compose.yml

```diff
@@ -18,11 +18,12 @@ services:
       - media_volume:/experimenter/experimenter/media
       - /experimenter/experimenter/legacy/legacy-ui/core/.cache/
       - /experimenter/experimenter/legacy/legacy-ui/core/node_modules/
+      - /experimenter/experimenter/theme/static_src/node_modules/
       - /experimenter/experimenter/nimbus-ui/node_modules/
       - /experimenter/experimenter/served/
       - /experimenter/node_modules/
       - ${GOOGLE_ADC_FILE}:${GOOGLE_APPLICATION_CREDENTIALS}:ro
-    command: bash -c "python bin/setup_kinto.py;/experimenter/bin/wait-for-it.sh db:5432 -- python /experimenter/manage.py runserver 0:7001"
+    command: bash -c "python bin/setup_kinto.py;/experimenter/bin/wait-for-it.sh db:5432 -- python /experimenter/manage.py runserver 0:7001 & cd /experimenter/experimenter/theme/static_src && yarn run dev"
```

Comment:
I'd love another pair of eyes on this since I'm not very familiar with the docker compose file

Reply:
This I changed from `&&` to `&` as the second one didn't execute because the `runserver` doesn't terminate. So to run both  commands I used the background operator `&` , [here's](https://stackoverflow.com/questions/3683910/executing-shell-command-in-background-from-script) a reference

Reply:
Ah yes good point, thanks for clarifying!

Does it make more sense to run the frontend in a separate container? Feels odd to background one process so you can run another in the docker context, but maybe this is normal.

If you keep it all in one container, consider putting each command on its own line for readability purposes.